django/db/models/sql/compiler.py

```diff
@@ -1627,6 +1630,9 @@ def explain_query(self):
                 else:
                     yield value
 
+    def comments_sql(self):
+        return " ".join([f"/* {comment} */" for comment in self.query.comments])
```

Comment:
Can we avoid the join? Callers should all be doing it already.

Reply:
DONE